_includes/pages/get-involved.html

```diff
@@ -172,7 +172,7 @@ <h3 class="is-size-3">{{ "Project Structure" | i18n }}</h3>
 				{{ "For more information, see the
 				<a href='https://docs.luanti.org/about/glossary/'>glossary</a> or
 				<a href='https://docs.luanti.org/for-engine-devs'>engine overview</a>
-				developer wiki pages." | i18n }}
+				developer documentation pages." | i18n }}
```

Comment:
IMO the end of this sentence can be shortened to just "[...], see the glossary or engine overview."